app/src/main/java/stage/hoogtebepaling5/SplashActivity.java

```diff
@@ -0,0 +1,38 @@
+package stage.hoogtebepaling5;
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+
+public class SplashActivity extends Activity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        // Creates the splashscreen content from the splashscreen.xml file in /layout
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.splashscreen);
+
+        Thread timerThread = new Thread(){
+            public void run(){
+                try{
+                    //goes on for two seconds
+                    sleep(1000);
```

Comment:
Wil je waarschijnlijk niet in de echte app versie doen, voor testen is het prima? op iOS is het verboden om een wachttijd op de spashscreen te hebben :P